L/Libepoxy/build_tarballs.jl

```diff
@@ -4,25 +4,47 @@ using BinaryBuilder
 
 name = "Libepoxy"
 version = v"1.5.10"
+# We bumped the version number to build for riscv64
+ygg_version = v"1.5.11"
 
 # Collection of sources required to build Libepoxy
 sources = [
-    ArchiveSource("https://github.com/anholt/libepoxy/archive/refs/tags/$(version).tar.gz",
-                  "a7ced37f4102b745ac86d6a70a9da399cc139ff168ba6b8002b4d8d43c900c15")
+    GitSource("https://github.com/anholt/libepoxy", "c84bc9459357a40e46e2fec0408d04fbdde2c973"),
+    DirectorySource("bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libepoxy-*/
+cd ${WORKSPACE}/srcdir/libepoxy
+
+# meson unconditionally (?) adds the command line argument
+# `-Werror=unused-command-line-argument`. That would be fine, except
+# that we (the `cc` script) add `-Wl,-sdk_version,11.0` on Darwin,
+# which is actually unused when meson checks for supported compiler
+# options. This means that each such check fails, and meson won't use
+# any options. That's mostly fine, except for the option
+# `-Wno-int-conversion` which is required by the source code.
+#
+# We thus use a wrapper for the `cc` script to filter out the option
+# `-Werror=unused-command-line-argument`. This should arguably either
+# happen by default, or `cc` should only add the
+# `-Wl,-sdk_version,11.0` when the linker is actually called.
```

Comment:
You're welcome to fix this in https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/45acaa5c78a091ef75e5daa5de7a8676600509a5/src/Runner.jl